### PR TITLE
Swaps: Alpha 2

### PR DESCRIPTION
**Pull request by wachunei**

### CODE REVIEW

File: ios/MetaMask.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -847,7 +847,7 @@
 				CODE_SIGN_ENTITLEMENTS = MetaMask/MetaMaskDebug.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Developer";
 				CODE_SIGN_STYLE = Manual;
-				CURRENT_PROJECT_VERSION = 564;
+				CURRENT_PROJECT_VERSION = 565;
```
Comment (estebanmino) [inline ios/MetaMask.xcodeproj/project.pbxproj:850]:
```suggestion
				CURRENT_PROJECT_VERSION = 566;
```

File: ios/MetaMask.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -910,7 +910,7 @@
 				CODE_SIGN_ENTITLEMENTS = MetaMask/MetaMask.entitlements;
 				CODE_SIGN_IDENTITY = "iPhone Distribution";
 				CODE_SIGN_STYLE = Manual;
-				CURRENT_PROJECT_VERSION = 564;
+				CURRENT_PROJECT_VERSION = 565;
```
Comment (estebanmino) [inline ios/MetaMask.xcodeproj/project.pbxproj:913]:
```suggestion
				CURRENT_PROJECT_VERSION = 566;
```

**Comment by estebanmino** [comment]
reminder to turn the feature off when we merge this

**Review by ibrahimtaveras00 — APPROVED** [review]
Alpha QA Passed ✅ 
ready for beta

**Pull request merged**